pytition/petition/templates/petition/detail2.html

```diff
@@ -1,268 +0,0 @@
-    <meta name="viewport" content="width=device-width"/>
-    <title>{% autoescape off %}{{ petition.raw_title }}{% endautoescape %}</title>
-    <meta name="description" content="{% autoescape off %}{{ petition.raw_title }}{% endautoescape %}"/>
-
-    <meta property="og:title" content="{% autoescape off %}{{ petition.raw_title }}{% endautoescape %}"/>
-    {% if petition.twitter_description %}
-        <meta property="og:description"
-              content="{% autoescape off%}{{ petition.raw_twitter_description }}{% endautoescape %}"/>
-    {% else %}
-        <meta property="og:description" content="{% autoescape off %}{{ petition.raw_text }}{% endautoescape %}"/>
-    {% endif %}
-    <meta property="og:type" content="website"/>
-    <meta property="og:url" content="https://petition.antipub.org/petition/{{ petition.id }}/"/>
-    <meta property="og:site_name" content="petition.antipub.org"/>
-    {% if petition.twitter_image %}
-        <meta property="og:image" content="{{ petition.twitter_image }}"/>
-    {% endif %}
-
-    <meta itemprop="name" content="{% autoescape off %}{{ petition.raw_title }}{% endautoescape %}"/>
-    {% if petition.twitter_description %}
-        <meta itemprop="description"
-              content="{% autoescape off %}{{ petition.raw_twitter_description }}{% endautoescape %}"/>
-    {% else %}
-        <meta itemprop="description" content="{% autoescape off %}{{ petition.raw_text }}{% endautoescape %}"/>
-    {% endif %}
-    <!-- FIXME: <meta itemprop="url" content=""/> //-->
-
-    <meta name="twitter:card" content="summary"/>
-    <meta name="twitter:creator" content="@yannsionneau"/>
-    <meta name="twitter:site" content="{% autoescape off %}{{ petition.org_twitter_handle }}{% endautoescape %}"/>
-    <meta name="twitter:title" content="{% autoescape off %}{{ petition.raw_title }}{% endautoescape %}"/>
-    {% if petition.twitter_description %}
-        <meta name="twitter:description"
-              content="{% autoescape off %}{{ petition.raw_twitter_description }}{% endautoescape %}"/>
-    {% else %}
-        <meta name="twitter:description" content="{% autoescape off %}{{ petition.raw_text }}{% endautoescape %}"/>
-    {% endif %}
-    {% if petition.twitter_image %}
-        <meta name="twitter:image" content="{{ petition.twitter_image }}"/>
-    {% endif %}
-    <!--<link rel="stylesheet" href="{% static "bootstrap.min.css" %}" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">-->
-    <link href="{% static "typeahead.css" %}" rel="stylesheet" type="text/css">
-    <link href="{% static "petition.css" %}" rel="stylesheet" type="text/css">
-    <style type="text/css">
-    {% if petition.linear_gradient_direction != petition.NO %}
-    body {
-        background: linear-gradient({{ petition.linear_gradient_direction}}, {{ petition.gradient_from }}, {{ petition.gradient_to }});
-    }
-    {% endif %}
-    .content {
-        min-height: 500px;
```

Comment:
This was put to fix the following issue: https://github.com/pytition/Pytition/issues/68
Maybe it was not the best way to fix the issue.
Do you think of a better way?

Reply:
Ah yes, I will check again : in my first version I used flexbox CSS3 properties and you just have to align content "stretch", but I used back absolute position for formular, so it is needed again.

Reply:
Back in commit 1b9369f :)

Reply:
Thanks!